.github/cspell.json

```diff
@@ -0,0 +1,32 @@
+{
+  "version": "0.1",
+  "language": "en",
+  "flagWords": [],
+  "ignoreWords": [
```

Comment:
Looking at https://cspell.org/configuration/, I believe this parameter should be called just `words`. Also, the `version` should be 0.2.